[2021-05-12 10:31:54,115] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: status_test.task1 2021-05-11T00:00:00+00:00 [queued]>
[2021-05-12 10:31:54,126] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: status_test.task1 2021-05-11T00:00:00+00:00 [queued]>
[2021-05-12 10:31:54,127] {taskinstance.py:1068} INFO - 
--------------------------------------------------------------------------------
[2021-05-12 10:31:54,127] {taskinstance.py:1069} INFO - Starting attempt 2 of 3
[2021-05-12 10:31:54,127] {taskinstance.py:1070} INFO - 
--------------------------------------------------------------------------------
[2021-05-12 10:31:54,311] {taskinstance.py:1089} INFO - Executing <Task(BashOperator): task1> on 2021-05-11T00:00:00+00:00
[2021-05-12 10:31:54,314] {standard_task_runner.py:52} INFO - Started process 2175663 to run task
[2021-05-12 10:31:54,322] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'status_test', 'task1', '2021-05-11T00:00:00+00:00', '--job-id', '55', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/task_status.py', '--cfg-path', '/tmp/tmp_0xpgpj3', '--error-file', '/tmp/tmpmhipqygd']
[2021-05-12 10:31:54,323] {standard_task_runner.py:77} INFO - Job 55: Subtask task1
[2021-05-12 10:31:54,469] {logging_mixin.py:104} INFO - Running <TaskInstance: status_test.task1 2021-05-11T00:00:00+00:00 [running]> on host sdn-k8s-b2-3-3.maas
[2021-05-12 10:31:54,841] {taskinstance.py:1281} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=davidwu5858@gmail.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=status_test
AIRFLOW_CTX_TASK_ID=task1
AIRFLOW_CTX_EXECUTION_DATE=2021-05-11T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-05-11T00:00:00+00:00
[2021-05-12 10:31:54,842] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2021-05-12 10:31:54,842] {bash.py:158} INFO - Running command: echo im task 1 !!!!!!!!!!!!!!!!!!!! || exit 1
[2021-05-12 10:31:54,851] {bash.py:169} INFO - Output:
[2021-05-12 10:31:54,853] {bash.py:173} INFO - im task 1 !!!!!!!!!!!!!!!!!!!!
[2021-05-12 10:31:54,853] {bash.py:177} INFO - Command exited with return code 0
[2021-05-12 10:31:54,973] {taskinstance.py:1185} INFO - Marking task as SUCCESS. dag_id=status_test, task_id=task1, execution_date=20210511T000000, start_date=20210512T103154, end_date=20210512T103154
[2021-05-12 10:31:55,253] {taskinstance.py:1246} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-05-12 10:31:55,258] {local_task_job.py:146} INFO - Task exited with return code 0
